web/themes/custom/pineapple/components/_comments-field.twig

```diff
@@ -25,8 +25,28 @@
  * @see comment_preprocess_field()
  */
 #}
-{% if not comments %}
-  <p>There are no comments on this content yet.</p>
+<section class="comments-wrapper">
+  {% if comment_form %}
+    <div class="comment-form-container">
+      {% include "@components/_heading.twig" with {
+        heading_level: 2,
+        heading_text: 'Leave a Comment'
+      } %}
+      {{ comment_form }}
+    </div>
+  {% endif %}
+
+  {% if comments %}
```

Comment:
You should also be checking if the label isn't hidden.

Reply:
Nah because I'm rendering it myself and I always want the label to show if there are comments.

Reply:
Hmmm, alarm bells are ringing can't quite tell why at the moment though.